ncm-metaconfig/src/main/perl/metaconfig.pod

```diff
@@ -66,40 +63,40 @@ it uses will generate them. The preamble attribute will be written out
 verbatim, before the contents is generated. No comment character is added,
 the user must specify this as part of the preamble string.
 
-=item * contents
+=item * C<contents>
 
 A free-form structure describing the valid entries for the
 configuration file. It is recommended to define another type for each
 config file, and bind it to these contents, to get the best
 validation.
 
-=item * element
+=item * C<element>
 
-Predefined conversions from L<EDG::WP4::CCM::TextRender>:
+Predefined conversions from C<< EDG::WP4::CCM::TextRender >>:
```

Comment:
I thought L<> was supposed to link to the EDG::WP4::CCM::TextRender page - does it not work? Or does C<<>> do the same thing?